apps/core/lib/core/snitch/variant.ex

```diff
@@ -8,21 +8,20 @@ defmodule Core.Snitch.Variant do
   @type t :: %__MODULE__{}
   schema "snitch_variants" do
     field(:sku, :string, default: "")
-    field(:weight, :integer, precision: 8, scale: 2, default: 0.0)
-    field(:height, :integer, precision: 8, scale: 2)
-    field(:width, :integer, precision: 8, scale: 2)
-    field(:depth, :integer, precision: 8, scale: 2)
+    field(:weight, :decimal, precision: 8, scale: 2, default: 0)
```

Comment:
What is the unit of these measurements? Please add a comment for this column in the table and note it in the module/type docs as well.